sdk/metricsadvisor/ai-metrics-advisor/recordings/node/metricsadvisoradministrationclient_datafeed_datafeed/recording_updates_data_feed_to_have_a_different_data_source_type.js

```diff
@@ -1,49 +1,5 @@
 let nock = require('nock');
 
-module.exports.hash = "b9a3cf10cfdec4fc77b26f76d0a2acd3";
+module.exports.hash = "ab0a998856c118deb560aa4e36c8a9d9";
 
-module.exports.testInfo = {"uniqueName":{},"newDate":{}}
```

Comment:
This file shouldn't change. Did the test fail?

Reply:
I see the `testInfo` got updated and the request-responses are deleted in the recording file.

Reply:
undid the changes. This file shouldn't change